### ISSUE TITLE
Home launch should check for 200 and 202 response

### ISSUE THREAD

**Issue by shenning00** [belltown/roku-develop#1]
The Roku will return a 202 if you try to launch an app that is already launched.

This may be a change in the later version of the firmware (7.5).

roku-develop.coffee line 472:

```javascript
      else if response.statusCode isnt 200
        atom.notifications.addWarning 'Bad response code ' +
                                      response.statusCode +
                                      ' from ' + ip, {dismissable: true}
```

Should include 202. Something like this:

```javascript
      else if [200,202].include(response.statusCode) == false 
        atom.notifications.addWarning 'Bad response code ' +
                                      response.statusCode +
                                      ' from ' + ip, {dismissable: true}
```

Let me know if you would like me to do a PR.

**Comment by belltown** [comment]
I couldn't reproduce this on my Roku. However, I've updated the code to handle an HTTP status code of 202 in the same way as code 200 is handled.

Update your roku-develop Atom package to version 0.1.4 and let me know if the problem persists.

**Comment by shenning00** [comment]
Sweet. Looks like that fixed it. And thanks for the great Atom package. I don't think I would be messing with my Roku without it.

FYI: I found this same problem in the python-roku package a week ago. I have Roku4 7.5.0 firmware.

**Comment by belltown** [comment]
Thanks for confirming that the issue is fixed.

FYI: If you want to try out another of my Roku tools, look at VioletBug, [https://github.com/belltown/violetbug](https://github.com/belltown/violetbug). It's a graphical interface to the Roku Debugger that should make life easier that using the command line for debugging. I haven't officially announced VioletBug yet, but you're welcome to try it out.

**Comment by shenning00** [comment]
Cool - I'll try it out. I'm going to get spoiled without my telnet and makefile pushes.

Thanks again.

**Comment by belltown** [comment]
Closed and fixed with release of version 0.1.4: [4b0af12](https://github.com/belltown/roku-develop/commit/4b0af12d544a2c67ac5027dcb96797851c192caf)
